wpilibc/athena/include/DriverStation.h

```diff
@@ -68,6 +68,7 @@ class DriverStation : public SensorBase, public RobotStateInterface {
   Alliance GetAlliance() const;
   int GetLocation() const;
   void WaitForData();
+  bool WaitForData(long long timeout);
```

Comment:
Time values elsewhere in WPILib are all expressed in seconds, so recommend making this a double seconds value.